internal/testing/releasehelper.go

```diff
@@ -0,0 +1,143 @@
+// Copyright 2019 The Go Cloud Development Kit Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Helper tool for creating new releases of the Go CDK.
+// Run from the repository root:
+// $ go run internal/testing/releasehelper.go <flags>
```

Comment:
Can you expand this help a bit (or, probably better, add `help` to the command itself)? IIUC, `replace` is going to add `replace` directives to all submodules, pointing `gocloud.dev` to `../..` or whatever. "Add replace directives to root" makes it sound like it will modify root, instead of the inverse.

Reply:
Done